module/behaviour/planning/FixedWalk/src/FixedWalk.cpp

```diff
@@ -102,7 +102,7 @@ namespace planning {
     std::unique_ptr<WalkCommand> FixedWalk::getWalkCommand(const FixedWalkCommand::WalkSegment& segment, NUClear::clock::duration t, const Sensors&){
      	double timeSeconds = std::chrono::duration_cast<std::chrono::seconds>(t).count();
     	Eigen::Vector2d directionInOriginalCoords = (segment.curvePeriod != 0 ? Rotation2D::createRotation(2 * M_PI * timeSeconds / segment.curvePeriod) : Eigen::Matrix<double, 2, 2>::Identity() ) * segment.direction;
-    	Eigen::Vector2d direction =  arma::normalise(directionInOriginalCoords);
+    	Eigen::Vector2d direction =  directionInOriginalCoords.normalize();
```

Comment:
Most of these should be .normalized() not .normalize(). One modifies and returns void, the other returns the modified value